yt_dlp/__init__.py

```diff
@@ -621,13 +621,13 @@ def get_postprocessors(opts):
             'api': opts.sponsorblock_api,
             'when': 'after_filter',
         }
-    if opts.convertsubtitles:
+    if opts.convertsubtitles and opts.convertsubtitles != 'none':
```

Comment:
I could also approach this with something like:

```python
if opts.convertsubtitles == 'none':
  opts.convertsubtitles = None
```

earlier in the file. This would also remove the need for my regex changes, but arguably it makes the code harder to follow. Let me know your preference!

Reply:
i think doing
```py
if opts.convertsubtitles == 'none':
  opts.convertsubtitles = None
```
for both should be the easiest